src/Traversal/Sdk/Sdk.targets

```diff
@@ -77,7 +88,7 @@
 
   <ItemGroup>
     <ProjectReference Update="@(ProjectReference)"
-                      Properties="$(TraversalGlobalProperties)" />
+                      AdditionalProperties="$(TraversalGlobalProperties)" />
```

Comment:
Not sure if `TraversalGlobalProperties` is supposed to rewrite all properties, or just add themselves among the possibly other existing global properties